src/server.c

```diff
@@ -3470,28 +3481,21 @@ void call(client *c, int flags) {
     dirty = server.dirty;
     incrCommandStatsOnError(NULL, 0);
 
-    const long long call_timer = ustime();
-
-    /* Update cache time, and indicate we are starting command execution.
-     * in case we have nested calls we want to update only on the first call */
-    if (server.execution_nesting++ == 0) {
-        updateCachedTimeWithUs(0,call_timer);
-        server.cmd_time_snapshot = server.mstime;
-        c->flags |= CLIENT_EXECUTING_COMMAND;
-    }
+    enterExecutionUnit();
+    const long long call_timer = server.ustime;
```

Comment:
so now `call_timer` may be outdated.
the previous code made sure to always call `ustime()` and get a fresh `call_timer`, and yet, avoid getting the system time twice, by passing that time to `updateCachedTimeWithUs`.
i think we must sort things out to get that functionality back.

Reply:
Ohh you are right, this is used to check the time it took to run the command. I will fix it.